### PR TITLE
Add xh as httpie alternative

### PR DESCRIPTION
**Pull request by jihchi**
> HTTPie is a user-friendly command-line HTTP client for the API era.

> xh is a friendly and fast tool for sending HTTP requests. It reimplements as much as possible of HTTPie's excellent design.

[asciicast](https://asciinema.org/a/390748)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,6 +118,10 @@ I renamed the repository to "Awesome Alternatives in Rust". The original name wa
 * [bottom](https://github.com/ClementTsang/bottom) - Yet another cross-platform graphical process/system monitor.
 * [ytop](https://github.com/cjbassi/ytop) (no longer maintained) - A TUI system monitor written in Rust
 
+#### [httpie](https://httpie.io/)
```
Comment (TaKO8Ki) [inline README.md]: Could you change this url to GitHub repository url?

```suggestion
#### [httpie](https://github.com/httpie/httpie)
```

File: README.md [hunk 2]
```diff
@@ -118,6 +118,10 @@ I renamed the repository to "Awesome Alternatives in Rust". The original name wa
 * [bottom](https://github.com/ClementTsang/bottom) - Yet another cross-platform graphical process/system monitor.
 * [ytop](https://github.com/cjbassi/ytop) (no longer maintained) - A TUI system monitor written in Rust
 
+#### [httpie](https://github.com/httpie/httpie)
+
+* [xh](https://github.com/ducaale/xh) - Friendly and fast tool for sending HTTP requests
```
Comment (TaKO8Ki) [inline README.md]: Could you follow the alphabet order? In this case, you should add httpie below hexdump.

Comment (jihchi) [reply README.md]: Done. fixed in commit 920c9b8252552d3503fa26e96c2896298b3e256c

Comment (TaKO8Ki) [reply README.md]: Thank you!

**Review by TaKO8Ki — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**